### ISSUE TITLE
Sort order does not place "You're here" first

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/prox#174]
I opened the app today and "The Bay Club at Waikoloa Beach Resort" was the first card shown despite having a 2min walking time. "Kamuela Provision Company", which has normally been first, is second and still has "You're here"

Questions:
* Can multiple cards have "You're here"? Should they be able to?
* How do we sort cards? By distance or travel times?

**Comment by mcomella** [comment]
> Should multiple cards be able to have "You're here"?

@antlam, what do you think?

**Comment by mcomella** [comment]
fwiw, it's slightly easier for us if they can.

**Comment by antlam** [comment]
@mcomella having multiple cards show "You're here" is interesting. But it can create a lot of confusion for our users right now. I think we should avoid this for Kona ATM.

> Can multiple cards have "You're here"? Should they be able to?

Not right now. This should always be the first card.

> How do we sort cards? By distance or travel times?

Travel times

**Comment by mcomella** [comment]
@antlam When we sort by travel times, should we sort by their walking travel times or overall travel time? I'm afraid with the latter, we'd get roughly one "3min walk", a lot of "3min drive", then a regular distribution of "10min walk/drive".

**Comment by antlam** [comment]
@mcomella I was talking to Brad and Sebastian about this in Taipei. 

Brad suggested an approach like this, which I think makes a lot of sense: 

1. Assume an inherent additional "lag time/cost" of 15 minutes to all Driving times
2. Sort all places in the backend by this "Total travel time" (Driving time + 15min) 
3. Display using "Walking time" IF a place is less than 15 minutes walking time away, display this in "# min walk".
4. Display using "Driving time" everywhere else, display this in "# min drive". NOT "(# + 15) min drive".

Note: We will still have a scenario where we see "1 min walk", "3 min walk", "3 min drive", "4 min drive", but I don't see why that's that's a huge issue.

Does this make sense to you? This way, the first couple results should still be "Walking times" and those would be things you could reach immediately within 15 minutes. Anything further, we'd display Driving times.

I based 15 minutes on how far an average person walks a day (from a quick googling - https://www.reference.com/health/many-miles-average-person-walk-per-day-d9d3aa8ef93e9ac3) and how long that would approximately take.

**Comment by antlam** [comment]
@mcomella looking back, I'm not sure if some of that ^ may be out of scope for this issue. Feel free to open another issue if it is. Just copy over what we're discussing above.

**Comment by mcomella** [comment]
Blocked on my travel times work (e.g. #184) but not going to have time to tackle this with that.
